tuplex/core/src/physical/ResolveTask.cc

```diff
@@ -228,8 +228,21 @@ namespace tuplex {
 
         // needs to be put into separate list of python objects...
         // save index as well to merge back in order.
-        assert(_rowNumber >= _numUnresolved);
-        _py_nonconfirming.push_back(std::make_tuple(_rowNumber - _numUnresolved, out_row));
+        assert(_currentRowNumber >= _numUnresolved);
+        auto pickledObject = python::pickleObject(python::getMainModule(), out_row);
+        auto pyObjectSize = pickledObject.size();
+        auto bufSize = 4 * sizeof(int64_t) + pyObjectSize;
+
+        uint8_t *buf = new uint8_t[bufSize];
+        auto ptr = buf;
+        *((int64_t*)ptr) = _currentRowNumber - _numUnresolved; ptr += sizeof(int64_t);
+        *((int64_t*)ptr) = ecToI64(ExceptionCode::PYTHON_PARALLELIZE); ptr += sizeof(int64_t);
+        *((int64_t*)ptr) = -1; ptr += sizeof(int64_t);
+        *((int64_t*)ptr) = pyObjectSize; ptr += sizeof(int64_t);
+        memcpy(ptr, pickledObject.c_str(), pyObjectSize);
+        rowToMemorySink(owner(), _fallbackSink, Schema(Schema::MemoryLayout::ROW, python::Type::makeTupleType({python::Type::STRING})),
```

Comment:
this won;t throw an exception, right? else, this may intropduce a memory leak...